brave/brave-core
# Replace Wallet onboarding conversion P3A question with a more precise question

Resolves brave/brave-browser#26011

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

0. Start with fresh profile, ensure metric does not exist on local state page.
1. Open Wallet onboarding page and exit. Ensure 0 is recorded for the metric
2. Create a new wallet, take note of the recovery phrase. Ensure 1 is recorded for the metric.
3. Add/import a secondary wallet. Ensure the metric value is the same.
4. Lock the wallet and restore using the recovery phrase. Ensure the metric value is the same.
5. Reset the wallet, restore the wallet using the recovery phrase. Ensure 0 is recorded before recovery and 2 is recorded after recovery.
